src/Runtime.Base/src/System/Runtime/RuntimeExports.cs

```diff
@@ -79,6 +79,19 @@ public static IntPtr RhHandleAllocVariable(object value, uint type)
             return h;
         }
 
+#if CORERT
```

Comment:
It would be good to have a comment here as to what you think about this #ifdef -- will it be permanent or does it represent a TODO?